core/neat/settings/_settings.scss

```diff
@@ -0,0 +1,20 @@
+/// Neat default grid.
+///
+/// @type map
+///
+/// @property {number (unitless)} columns 12
+///   Number of grid columns
+///
+/// @property {number (with unit)} gutter 20px
+///   Grid gutter width.
+///
+/// @property {number (unitless)} max-width 1200px
+///   Grid container's max-width.
+///
+/// @access private
+
+$neat-default-grid: (
+  columns: 12,
+  gutter: 20px,
+  max-width: 1200px,
```

Comment:
Is there a reason we want `max-width` to be embedded as part of the grid’s parameters? What’s the benefit of that, versus me having my own containers with `max-width` set (or _not_ set!)?